packages/kbn-eslint-plugin-i18n/jest.config.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
   preset: '@kbn/test/jest_node',
   rootDir: '../..',
   roots: ['<rootDir>/packages/kbn-eslint-plugin-i18n'],
+  globals: {
```

Comment:
Added as a workaround to keep using `--watch` after the Node 20 upgrade. More details here: https://elastic.slack.com/archives/C5UDAFZQU/p1704454145329549